Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'hiredis'
 gem 'http_accept_language'
 gem 'httparty'
 gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'master'
-gem 'identity-telephony', github: '18f/identity-telephony', tag: 'v0.1.1'
+gem 'identity-telephony', github: '18f/identity-telephony', tag: 'v0.1.2'
```

Comment:
This bump includes https://github.com/18F/identity-telephony/pull/27 which adds a fake `message_id` to the fake SMS sender